pkgs/by-name/ya/yaak/package.nix

```diff
@@ -95,13 +106,25 @@ rustPlatform.buildRustPackage (finalAttrs: {
       ln -s ${protobuf}/include src-tauri/vendored/protoc/include
     '';
 
+  tauriBuildFlags = [
```

Comment:
This is what they are using in the CI, if we don't set this it will be complaining about not being able to load the updater plugin